content/en/docs/How Tos/create_filter.md

```diff
@@ -21,11 +21,16 @@ the available filters in [`Components`](https://pkg.go.dev/github.com/AdRoll/bak
 
 ## The Filter interface
 
-The [Filter interface](https://pkg.go.dev/github.com/AdRoll/baker#Filter) determines the functions
-that a filter must implement. The interface is quite simple and contains only two functions:
-`Process(l Record, next func(Record))` and `Stats() FilterStats`:
+New *Filter* components need to implement the [Filter interface](https://pkg.go.dev/github.com/AdRoll/baker#Filter).
```

Comment:
I'd use `Filter` or **Filter**, but not *Filter* that, if I'm not wrong, we're not using anywhere else